packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1052,7 +1052,8 @@ enroll.mdm.copyLink = Copy link to clipboard
 enroll.mdm.copyLink.success = Link copied.
 enroll.mdm.step1 = Tap the Copy Link button below.
 enroll.mdm.step2 = On this device, open your browser, then paste the copied link into the address bar.
-enroll.mdm.step3 = Follow the instructions in your browser to set up <span class="semi-strong">{0}</span>, then try accessing this app again.
+enroll.mdm.step3 = Follow the instructions in your browser to set up <span class="semi-strong">{0}</span>.
+enroll.mdm.step4 = Logout and re-login and then try accessing the app again.
```

Comment:
This change is going to force our translations to be out-of-sync. As a result, non-English users will see the old `enroll.mdm.step3` message **and** the English (or `L10N_ERROR`) value of `enroll.mdm.step4`.

Please follow [this process](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/741293778/i18n+dev+process+From+Oct+2019+onwards#If-translations-already-exist-for-the-key%3A).